javaslang/src/test/java/javaslang/collection/BitSetTest.java

```diff
@@ -524,4 +504,33 @@ public void shouldConvertToSortedSetWithoutComparatorOnComparable() {
             assertThat(set).isEqualTo(TreeSet.of(0, 1, 3, 7, 15));
         }
     }
+
+    // -- classes
+
+    private static class Mapper<T> implements Serializable {
+
+        private static final long serialVersionUID = 1L;
+
+        private final java.util.Map<Integer, T> fromIntMap = new java.util.HashMap<>();
+        private final java.util.Map<T, Integer> toIntMap = new java.util.HashMap<>();
+        private int nextValue = 0;
```

Comment:
\[Checkstyle\] ERROR: Variable 'nextValue' explicitly initialized to '0' \(default value for its type\)\.